### ISSUE TITLE
has method does not work in postgresql

### ISSUE THREAD

**Issue by happytor** [catfan/Medoo#127]
because select exists .... will return 't' or 'f'

```
$result=$this->query($this->select_context($table, $join, $column, $where, 1))->fetchColumn();
return $result === '1'||$result===1;
```

or

```
$result=$this->query('SELECT EXISTS(' . $this->select_context($table, $join, $column, $where, 1) . ')')->fetchColumn()
return $result === '1'||$result==='t';
```

**Comment by jjworld** [comment]
It appears that the fetchColumn() result is the integer 1 while the comparison is expecting a string with '1'. If the comparison is changed from === to == it seems to work fine.

**Comment by jimsproject** [comment]
is there any plan to solve this isue  for postgresql?
thank you

**Comment by catfan** [comment]
Fixed 9f41dfa29e9d91d8c1d29e4219b02652d6ac4461
